docs/src/howto-guides/002_installation.md

```diff
@@ -10,13 +10,56 @@ curl -fsSL https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/get-thi
 
 You can execute that command on your device and it will do all required steps for an initial setup.
 
-If you prefer to have a little more control over the installation or the script did not work for you, please go on with the following steps.
+> Note: If you want to get a specific version, add `<version>` at the end like below. 
+> `<version>` is consist of 3 digits, e.g. `0.7.3`.
+> ```shell
+> curl -fsSL https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/get-thin-edge_io.sh | sudo sh -s <version>
+> ```
+
+If you prefer to have a little more control over the installation or the script did not work for you,
+please go on with the [manual installation steps](#manual-installation-steps).
+
+## Upgrade thin-edge.io with get-thin-edge_io.sh script
+
+If you already have `thin-edge.io` on your device, to upgrade `thin-edge.io`,
+the easiest way is to use the same script as the installation. Follow the steps below.
+there is no need to remove old version.
+
+> Note: To successfully upgrade `thin-edge.io` all thin-edge.io components **must be stopped**. The components are:
+> `tedge-mapper-c8y`, `tedge-mapper-az`, `tedge-mapper-collectd`, `tedge-agent`, `c8y-log-plugin`, `c8y-configuration-plugin`.
+>
+> To stop `tedge-mapper-c8y`, `tedge-agent`, `tedge-mapper-az`, you can simply run the commands below.
+> 
+> ```shell
+> sudo tedge disconnect c8y
+> sudo tedge disconnect az
+> ```
+> 
+> To stop each component one by one, this is an example how to stop them with `systemctl`:
+> 
+> ```shell
+> systemctl stop tedge-mapper-c8y
+> systemctl stop tedge-agent
+> systemctl stop c8y-log-plugin
+> ```
```

Comment:
Why is this a quote?

Reply:
For this, I just kept the original document style. No big reason.